src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowProvider.cs

```diff
@@ -79,7 +80,8 @@ public IVsInteractiveWindow Create(int instanceId)
         {
             var evaluator = CreateInteractiveEvaluator(_vsServiceProvider, _classifierAggregator, _contentTypeRegistry, _vsWorkspace);
 
-            var vsWindow = _vsInteractiveWindowFactory.Create(Id, instanceId, Title, evaluator, 0);
+            // ForceCreate means that the window should be created if the persisted layout indicates that it is visible.
```

Comment:
What does visible mean here? If it is open but the interactive window tab is not selected then is it considered visible? If yes then this seems right.